src/simulations/local_elec.py

```diff
@@ -556,8 +559,15 @@ def policy_to_rewardvar(
             bool_penalty, date, loads, E_req_only, h, last_step, home_vars)
 
         if self.prm['grd']['manage_voltage']:
+            start = time.time()
             hourly_line_losses, voltage = self.network.pf_simulation(home_vars['netp'])
+            end = time.time()
```

Comment:
so because `pf_simulation` is not only called here but also in another place -- I would maybe time this in the `network.pf_simulation` method each time it is called at the start and end of the method. Append a network attribute list each time it is called. Then in `record.save(end_of='end')` you can save the mean and std of all the values in that list `self.explorer.env.network.times_pandapower_simulations`.